services/frontend/src/components/formItems/DatePickerField.jsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { makeStyles, ThemeProvider } from "@material-ui/styles";
+import { createMuiTheme } from "@material-ui/core";
+import DayJSUtils from "@date-io/dayjs";
```

Comment:
date-io might not have been installed properly. My compiler can't find it.

Reply:
Not sure what caused this... 

I restarted my local server and it got rid of the compiler error that said: "Module not found". 

I removed the date-io package and re-added it with its dependencies. Please let me know if this did anything